pkgs/http/lib/src/browser_client.dart

```diff
@@ -27,18 +31,20 @@ BaseClient createClient() {
 }
 
 /// A `package:web`-based HTTP client that runs in the browser and is backed by
-/// [XMLHttpRequest].
+/// `window.fetch`.
+///
+/// This client inherits some limitations of `window.fetch`.
+/// - [BaseRequest.persistentConnection];
+/// - Setting [BaseRequest.followRedirects] to `false` will cause
+/// `ClientException` when redirect is encountered;
```

Comment:
```suggestion
///   `ClientException` when a redirect is encountered;
```

Reply:
(or does the markdown get formatted correctly without the indent?

Reply:
It does get formatted correctly but I applied your suggestion because it makes things easier to read.